src/modules/src/controller_lee_payload.c

```diff
@@ -477,9 +514,25 @@ PARAM_ADD(PARAM_FLOAT, offsetx, &g_self.offset.x)
 PARAM_ADD(PARAM_FLOAT, offsety, &g_self.offset.y)
 PARAM_ADD(PARAM_FLOAT, offsetz, &g_self.offset.z)
 
+PARAM_ADD(PARAM_FLOAT, n1, &g_self.n1.x)
+PARAM_ADD(PARAM_FLOAT, n1, &g_self.n1.y)
+PARAM_ADD(PARAM_FLOAT, n1, &g_self.n1.z)
+PARAM_ADD(PARAM_FLOAT, n2, &g_self.n2.x)
+PARAM_ADD(PARAM_FLOAT, n2, &g_self.n2.y)
+PARAM_ADD(PARAM_FLOAT, n2, &g_self.n2.z)
+
+
+PARAM_ADD(PARAM_FLOAT, radius, &g_self.radius)
+
 
 //For the QP 
 PARAM_ADD(PARAM_FLOAT, value, &g_self.value)
+PARAM_ADD(PARAM_FLOAT, mu1x, &g_self.mu1.x)
+PARAM_ADD(PARAM_FLOAT, mu1y, &g_self.mu1.y)
+PARAM_ADD(PARAM_FLOAT, mu1z, &g_self.mu1.z)
+PARAM_ADD(PARAM_FLOAT, mu2x, &g_self.mu2.x)
+PARAM_ADD(PARAM_FLOAT, mu2y, &g_self.mu2.y)
+PARAM_ADD(PARAM_FLOAT, mu2z, &g_self.mu2.z)
```

Comment:
Remove. If you want to be able to look at them, you would need to add them to logging, not to params